internal/ad/ad_test.go

```diff
@@ -626,6 +626,20 @@ func TestGetPolicies(t *testing.T) {
 			turnKrb5CCRO:       true,
 			wantErr:            true,
 		},
+		"Unsupported type for unfiltered entry": {
```

Comment:
Actually, I think another test would be beneficial. Something like "Ignore Errors on non Ubuntu keys" or any better wording. I agree this is a superset because currently, this is the same code skipping them first, but we never know what could happen in the future… (like, we start supporting some Windows keys…). I would thus decouple those and revert the .pol file. WDYT?